server/endpoints/Review.ts

```diff
@@ -0,0 +1,54 @@
+import { body } from "express-validator";
+import { getCrossListOR } from "common/CourseCard";
+import { Endpoint } from "../endpoints";
+import { Reviews, Classes } from "../dbDefs";
+
+interface CourseId {
+  courseId: string;
+}
+
+export const getCourseById: Endpoint<CourseId> = {
```

Comment:
It would be nice to have a little bit of documentation here. Just a couple of words should do, I think.

Reply:
I copied the comments from the original function but i think you are right. Not sure why "local" was used. I can remove it.

Reply:
You are correct Julian, "Local" is wrong in this context. Generally speaking it should be pulling from staging/prod, which is not local.